taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -14,46 +15,104 @@ import ListAllEntryRelationships from "./ListAllEntryRelationships";
+        fetch(url, {
+            method : 'POST',
+            headers: {"Content-Type" : "application/json"},
+            body: JSON.stringify(data)
+        }).then(() => {
+            if (isEntry) {
+                fetch(url+'children/', {
+                    method : 'POST',
+                    headers: {"Content-Type" : "application/json"},
+                    body: JSON.stringify(updateChildren)
+                }).then(() => {
+                    setOpen(true);
+                }).catch((errorMessage) => {
+                    console.log(errorMessage);
+                })
+            }
+        }).catch((errorMessage) => {
+            console.log(errorMessage);
+        })
     }
     return ( 
         <Box className="node-attributes">
             {/* Based on isEntry, respective components are rendered */}
             { isEntry ? 
                 <Box>
                     { !!nodeObject &&
-                        <>  <ListAllEntryRelationships url={url+'parents'} title={'Parents'} />
-                            <ListAllEntryRelationships url={url+'children'} title={'Children'} />
-                            <ListTranslations nodeObject={nodeObject} setNodeObject={setNodeObject} /> 
-                            <ListAllEntryProperties nodeObject={nodeObject} setNodeObject={setNodeObject} /> </> }
+                        <>  <ListEntryParents url={url+'parents'} />
+                            <ListEntryChildren url={url+'children'} setUpdateNodeChildren={setUpdateChildren} />
+                            <ListAllEntryProperties nodeObject={updatedNodeObject} setNodeObject={setUpdatedNodeObject} originalNodeObject={nodeObject} /> </> }
                 </Box> :
-                <>  <ListAllNonEntryInfo nodeObject={nodeObject} id={id} setNodeObject={setNodeObject} /> </>
+                <>  <ListAllNonEntryInfo nodeObject={updatedNodeObject} id={id} setNodeObject={setUpdatedNodeObject} originalNodeObject={nodeObject} /> </>
             }
+            {/* Button for submitting edits */}
+            <Button
+                variant="contained"
+                onClick={handleSubmit}
+                sx={{ml: 4, mt:2, width: 130}}>
+                    Submit
+            </Button>
+            {/* Dialog box for acknowledgment of update */}
+            <Dialog
+                open={open}
+                aria-labelledby="alert-dialog-title"
+                aria-describedby="alert-dialog-description"
+            >
+                <DialogTitle id="alert-dialog-title">
+                {"Your edits have been saved!"}
+                </DialogTitle>
+                <DialogContent>
+                <DialogContentText id="alert-dialog-description">
+                    The node {id} has been successfully updated.
+                </DialogContentText>
+                </DialogContent>
+                <DialogActions>
+                <Button sx={{fontFamily: 'Roboto, Helvetica, Arial, sans-serif'}} onClick={handleBack}>
```

Comment:
instead of navigate, better use a `Link`

Reply:
why these have a different font family?

Reply:
The button text wasn't clear. I'll add it to the ThemeProvider.